app/controlplane/internal/data/ent/migrate/migrations/20240610060712.sql

```diff
@@ -0,0 +1,6 @@
+-- Add default value to "name" column in "workflows" table if it is null
+UPDATE "workflows" SET "name" = CONCAT(split_part("id"::text, '-', 1)) WHERE "name" IS NULL;
+-- Create index "workflows_name_organization_id" to table: "workflows"
+CREATE UNIQUE INDEX "workflows_name_organization_id" ON "workflows" ("name", "organization_id") WHERE (deleted_at IS NULL);
```

Comment:
I am confused. Hasn't this already been done?

From db schema.

```
"workflow_name_organization_id" UNIQUE, btree (name, organization_id) WHERE deleted_at IS NULL
```

Reply:
Yes, they were there but I don't know why my schema was not registering those changes and I couldn't see the index that's why I created.

Doing a hard reset fixed the issue. Sorry for the noise.